examples/tutorials/credit/apply_policy_spark.py

```diff
@@ -0,0 +1,23 @@
+import cape_privacy as cape
+import pandas as pd
+from pyspark import sql
+
+# Set up your SparkSession as usual, but configure it for use with Cape.
+# We do this because some transformations expect Arrow to be enabled.
+sess = sql.SparkSession.builder \
+    .appName("cape.tutorial.maskPersonalInformation") \
+    .getOrCreate()
+sess = cape.spark.configure_session(sess)
+
+# Load a Spark DataFrame
+pdf = pd.read_csv('data/credit_with_pii.csv')
+pdf.Application_date = pd.to_datetime(pdf.Application_date)
+df = sess.createDataFrame(pdf)
```

Comment:
alternatively, you could use `sess.read.load` or `sess.read.csv` to be more "spark-ish"